### ISSUE TITLE
The application is not building for Windows

### ISSUE THREAD

**Issue by maxxxbar** [JetBrains/compose-jb#313]
```
    kotlin("jvm") version "1.4.21-2"
    id("org.jetbrains.compose") version "0.0.0-vsync-build30"
```
When assembling an exe or msi files, an error occurs
```
WARNING: Using incubator modules: jdk.incubator.jpackage
java.io.IOException: Command [light.exe, -nologo, -spdb, -ext, WixUtilExtension, -out, C:\Users\Maxim\AppData\Local\Temp\jdk.incubator.jpackage7221070195802763069\images\win-exe.image\example-0.1.msi, -sice:ICE27, -ext, WixUIExtension, -loc, C:\Users\Maxim\AppData\Local\Temp\jdk.incubator.jpackage7221070195802763069\config\MsiInstallerStrings_en.wxl, -b, C:\Users\Maxim\AppData\Local\Temp\jdk.incubator.jpackage7221070195802763069\config, C:\Users\Maxim\AppData\Local\Temp\jdk.incubator.jpackage7221070195802763069\wixobj\main.wixobj, C:\Users\Maxim\AppData\Local\Temp\jdk.incubator.jpackage7221070195802763069\wixobj\bundle.wixobj]in C:\Users\Maxim\AppData\Local\Temp\jdk.incubator.jpackage7221070195802763069\images\win-msi.image\example exited with 103 code

Execution failed for task ':packageExe'.
> Process 'command 'C:\Program Files\Java\jdk-15.0.2\bin\jpackage.exe'' finished with non-zero exit value 1
```

**Comment by olonho** [comment]
Could you please provide reproducer?

**Comment by AlexeyTsvetkov** [comment]
Possibly duplicate of https://github.com/JetBrains/compose-jb/issues/196

**Comment by maxxxbar** [comment]
> Could you please provide reproducer?

https://github.com/maxxxbar/NomenclatureConverter

**Comment by AlexeyTsvetkov** [comment]
The issue is fixed by the fix of #196
